README.md

```diff
@@ -84,12 +71,12 @@ Once the dependencies are installed, you can install the latest stable release o
 pip install deeprank2
 ```
 
-Alternatively, get all the new developments by cloning the repo and installing the code with
+Alternatively, get all the new developments by cloning the repo and installing the editable version of the package with
 
 ```bash
 git clone https://github.com/DeepRank/deeprank2
 cd deeprank2
-pip install -e ./
+pip install -e .
 ```
```

Comment:
it would be nice to also include the command for adding the test packages as well (I always forget the where it needs to go and which part needs to be in quotations): `pip install -e '.[test]'`

Reply:
Done, but JFYTK you can look at the action.yml file, and there you'll see how the test dependencies are added :)